src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -825,9 +825,7 @@ std::string ContextAttribute::check
 
   if ((compoundValueP != NULL) && (compoundValueP->childV.size() != 0))
   {
-    // FIXME P9: Use CompoundValueNode::check here and stop calling it from where it is called right now.
-    //           Also, change CompoundValueNode::check to return std::string
-    return "OK";
+    return compoundValueP->check();
```

Comment:
Nice to remove a P9 !!!
Perhaps we need to look at the FIXMEs a little more than we do ...
Right now it is closer to zero than to 0.001 ... 
It seems this PR did a bug fix about something we alreadsy knew, that we had in a FIXME P9.
A little bit sad, if it is so ,,,

NTC

Just a reflection